### DISCUSSION TITLE
How to use `--persist-to` wrangler command flag with remix cloudflare?

### DISCUSSION THREAD

**Discussion by samyakb** [remix-run/remix#9989] [Q&A] [answered]
I have seperate repo for workers and pages, but I want to share KV data between those projects. After reading docs, I found there is a flag for wrangler command as `--persist-to` to store local state in one place. But with remix run template, the local development command is using `remix vite:dev` . So I am confused where to use this flag ? I couldn't find anything to put in `wrangler.toml` as well

**Comment by andycrum** [comment, marked as answer]
@hmble The change will be in your vite.config.ts file. `cloudflareDevProxyVitePlugin` has a `persist` config option where you can pass in a relative `path` to the folder containing your data (I think mostly undocumented, I had to really dig to find it).

So part of your your vite config's `plugins` section might look like this:

```
plugins: [
  cloudflareDevProxyVitePlugin({
    getLoadContext,
    persist: {
      path: "../..../_local_db/v3",
    },
  }),
 ...(other stuff here)
```

**Reply by samyakb** [reply]
Thank you for this. This worked
